protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2314,6 +2327,31 @@ message GetActiveNetworkHistoryPeerAddressesResponse {
     repeated string ip_addresses = 1;
 }
 
+// Request to get the status of network history
+message NetworkHistoryStatusRequest {}
+
+// Response containing the status of network history
+message NetworkHistoryStatusResponse {
+    // The ipfs address of this node
+    string ipfs_address = 1;
+    // The swarm key
+    string swarm_key = 2;
+    // The swarm key seed
+    string swarm_key_seed = 3;
+    // The connected peers
+    repeated string connected_peers = 5;
```

Comment:
Any reason why we can't just return the bootstrap peers in the status response, instead of it having a dedicated endpoint?

Reply:
yeah, I wanted to encourage people not to use the same bootstrap peers set, but have it available if needed, so my thinking was to split this into a seperate call, and have the status display a randomised list of  the current active peers so that there is a better chance that the new nodes use this and don't all end up with the same bootstrap peers -  but at the same time it could be useful information to have available - I did toy with not exposing bootstrap peers at all.